### PR TITLE
Prepare repository to release WildFly 11

### PR DESCRIPTION
**Pull request by dgonzalezruiz**
**Description of the change**

This change adds support for WildFly 11.

**Benefits**

We will be able to support the must updated version of WildFly

**Possible drawbacks**

No drawbacks aFAIC

### CODE REVIEW

File: 11/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM bitnami/minideb-extras:jessie-r13
 MAINTAINER Bitnami <containers@bitnami.com>
 
+
```
Comment (sameersbn) [inline 11/Dockerfile]: please cleanup these newline characters

File: 11/Dockerfile [hunk 2]
```diff
@@ -9,6 +10,7 @@ ENV BITNAMI_IMAGE_VERSION=11.0.0-r0 \
 RUN install_packages libc6 libxext6 libx11-6 libxcb1 libxau6 libxdmcp6 libglib2.0-0 libfreetype6 libfontconfig1 libstdc++6 libgcc1 zlib1g libselinux1 libpng12-0 libexpat1 libffi6 libpcre3 libxml2 liblzma5
 
 # Additional modules required
+
```
Comment (sameersbn) [inline 11/Dockerfile]: this one as well

File: README.md [hunk 3]
```diff
@@ -26,7 +26,8 @@ services:
 
 # Supported tags and respective `Dockerfile` links
 
- - [`11`, `11.0.0-r0` (11/Dockerfile)](https://github.com/bitnami/bitnami-docker-wildfly/blob/0e0e99edc3b7238d5602b74a59f5fb20fe35782d/11/Dockerfile)
+
+ - [`11`, `11.0.0-r0`, `latest` (11/Dockerfile)](https://github.com/bitnami/bitnami-docker-wildfly/blob/master/11/Dockerfile)
```
Comment (sameersbn) [inline README.md]: The links should actually be pointing to the Dockerfile from the particular blob and not the master

**Comment by jotadrilo** [comment]
this PR does no longer apply since the `11` branch is already in master

**Pull request closed without merging**